Core.Tests/PrivatTransactionsImporterTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml;
+using Xunit;
+
+namespace Core.Tests
+{
+    public class PrivatTransactionsImporterTests
+    {
+        [Fact]
+        public void GetTransactionFromXml_WhenCorrectXml_ReturnTransaction()
+        {
+            var expectedTransaction = new Transaction("5555555555555555", "188888", DateTimeOffset.Parse("01/03/2020 18:58:00"), new Money(-599.03M, Currency.UAH),
+                new Money(1300.74M, Currency.UAH), "Wizz Air", "Payment");
+
+            var xmlContent = "<statement card=\"5555555555555555\" appcode=\"188888\" trandate=\"2020 - 01 - 03\" " +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'xmlContent'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)